demo/badge/badge.js

```diff
@@ -2,9 +2,20 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Icon from '../../src/Icon';
 import Badge from '../../src/Badge';
+import Textfield from '../../src/Textfield';
 
 class Demo extends React.Component {
+    state = {
```

Comment:
It isn't static, it's on instance, static should be prepended by `static` keyword :-)